Installation/CMakeLists.txt

```diff
@@ -526,7 +510,7 @@ message("== Detect external libraries ==")
 #  External libs configured when Qt5 lib of cgal are required
 #  Coin is used in KDS, but no FindCoin or FindCOIN exists
 #  There exists FindIPE, FindMKL, but they are only used to support supporting libs
-list (INSERT CGAL_SUPPORTING_3RD_PARTY_LIBRARIES 0 GMP MPFR ZLIB OpenGL LEDA MPFI RS RS3 OpenNL Eigen3 ESBTL Coin3D NTL IPE)
+list (INSERT CGAL_SUPPORTING_3RD_PARTY_LIBRARIES 0 LEDA )
```

Comment:
That variable `CGAL_SUPPORTING_3RD_PARTY_LIBRARIES` should disappear. I understand why we still use `CGAL_ESSENTIAL_3RD_PARTY_LIBRARIES`, but `CGAL_SUPPORTING_3RD_PARTY_LIBRARIES` should be useless, now.